frontend/src/components/homepage/components/search/search.tsx

```diff
@@ -0,0 +1,45 @@
+import Input from 'components/common/input/input';
+import { useAppForm } from 'hooks/hooks';
+import { DEFAULT_PODCAST_SEARCH_PAYLOAD } from './common/constants';
+import { IconName, InputType } from 'common/enums/ui/ui';
+import { FormEvent } from 'common/enums/enums';
+import { FieldValues, UnpackNestedValue } from 'react-hook-form';
+import styles from './styles.module.scss';
+
+type Props = {
+  onChange: (evt: UnpackNestedValue<FieldValues>) => void;
```

Comment:
```suggestion
  onChange: (evt: TYPE) => void;
```
add normal type here

Reply:
don't fixed

Reply:
can't change the type that some how connects to useForm